neurodsp/sim/cycles.py

```diff
@@ -0,0 +1,275 @@
+"""Simulating individual cycles, of different types."""
+
+import numpy as np
+from scipy.signal import gaussian, sawtooth
+
+from neurodsp.utils.checks import check_param
+from neurodsp.sim.info import get_sim_func
+from neurodsp.sim.transients import sim_synaptic_kernel
+
+###################################################################################################
+###################################################################################################
+
+def sim_cycle(n_seconds, fs, cycle_type, phase=0, **cycle_params):
+    """Simulate a single cycle of a periodic pattern.
+
+    Parameters
+    ----------
+    n_seconds : float
+        Length of cycle window in seconds.
+        This is NOT the period of the cycle, but the length of the returned array of the cycle.
+    fs : float
+        Sampling frequency of the cycle simulation.
+    cycle_type : {'sine', 'asine', 'sawtooth', 'gaussian', 'exp', '2exp'}
+        What type of cycle to simulate. Options:
+
```

Comment:
There is no docstring for `phase`.

Reply:
Ah, phase in this func was accidentally leftover from when I was playing with where to apply phase shifts. I've dropped it from this func, and removed it from the signature.